README.md

```diff
@@ -1,2 +1,140 @@
 # postgres-queries
-Query builder for PostgreSQL in Node.js, built for async
+
+Utility to generate and execute postgresql queries with ease.
+
+## Install
+
+`npm install --save postgres-queries`
+
+## Introduction
+Creating query and executing them are two separate concerns. And thus postgres-queries is divided in two parts:
+- The pool, that allows to connect to the postgres database and execute query.
+- The query builders (insertOne, selectOne, etc..) that allows to generate sql, and the corresponding parameters.
+
+## Documentation
+go to [https://marmelab.com/postgres-queries/](https://marmelab.com/postgres-queries/)
+
+## Pool
+Extend [node-pg-pool](https://github.com/brianc/node-pg-pool)
+Allow to connect to postgresql and execute query
+It adds:
+- namedQuery
```

Comment:
I suggest to add a link to the doc of the namedQuery function?
Or at least, explain what's its purpose.